src/components/Books.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Form from './form';
+import Book from './book';
+
+const Books = () => (
+  <>
+    <Book title="Book title" author="Book author" />
+    <Form />
+  </>
```

Comment:
- [ ] Instead of having to call every time Book, I'll suggest you use an array of list of books and  iterate through the list of books to get the details of each book(title,author,...)
Example below:
```
const Books = () => {
  const books = [
    {
      id: 1,
      title: 'Something',
      author: 'Something',
    },
    {
      id: 1,
      title: 'Something',
      author: 'Something',
    },
  ];
  // then you can have something like
  return (
    <>
      <ul>
        { books.map((book) => (<Book key={book.id} title={book.title} author={book.author} />))}
      </ul>
      <Form />
    </>
  );
};
export default Books;
```

Reply:
Hi @RolandM99 
As you may see very well, right there It's like I'm passing the props down to the child component. Those strings that I have put right there are only like a placeholder. because even as you may read in the project requirements it's mentioned that we keep a place where our book title and author will be put. this means when I will developing the app in the coming exercise those strings will be replaced with a specific element, that the user will be typing the input fields. (Either a state key-value if I choose to use the class comment or props If  I use functional components)

Reply:
- [ ] I suggested you use this approach in order to avoid redundancy and then use the best practices you learned about in the previous lessons. For now, when you want to add a book you have to repeat calling `Book`.